webapp/channels/src/components/channel_notifications_modal/components/expand_view.tsx

```diff
@@ -272,6 +312,72 @@ export default function ExpandView({
                 </fieldset>
             </>
             }
+            {(section === NotificationSections.DESKTOP) && memberNotifyLevel !== NotificationLevels.NONE &&
+
+            <>
+                <hr/>
+                <fieldset>
+                    <legend className='form-legend'>
+                        <FormattedMessage
+                            id='channel_notifications.sound'
+                            defaultMessage='Notification sound'
+                        />
+                    </legend>
+                    <div className='radio'>
+                        <label className=''>
+                            <input
+                                id='channelDesktopSoundOn'
+                                name='channelDesktopSound'
+                                type='radio'
+                                value={DesktopSound.ON}
+                                checked={memberDesktopSound === DesktopSound.ON}
+                                onChange={onChangeDesktopSound}
+                            />
+                            <FormattedMessage
+                                id='channel_notifications.sound.on.title'
+                                defaultMessage='On'
+                            />
+                        </label>
+                    </div>
+                    <div className='radio'>
+                        <label>
+                            <input
+                                id='channelDesktopSoundOff'
+                                name='channelDesktopSound'
+                                type='radio'
+                                value={DesktopSound.OFF}
+                                checked={memberDesktopSound === DesktopSound.OFF}
+                                onChange={onChangeDesktopSound}
+                            />
+                            <FormattedMessage
+                                id='channel_notifications.sound.off.title'
+                                defaultMessage='Off'
+                            />
+                        </label>
+                    </div>
+                    {memberDesktopSound === DesktopSound.ON && (!isDesktopApp() || (window.desktop && semver.gte(window.desktop.version || '', '4.6.0'))) &&
```

Comment:
why did we had to check for a specific desktop version here?

Reply:
we have removed them https://github.com/mattermost/mattermost-server/pull/23558/files

Reply:
I removed it :)